Assets/Scripts/Layers/LayerTypes/WMSLayerGameObject.cs

```diff
@@ -0,0 +1,67 @@
+using Netherlands3D.Twin.Layers.Properties;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Netherlands3D.Twin.Layers
+{
+    /// <summary>
+    /// Extention of LayerGameObject that injects a 'streaming' dataprovider WMSTileDataLayer
+    /// </summary>
+    public class WMSLayerGameObject : CartesianTileLayerGameObject, ILayerWithPropertyData
+    {
+        private WMSTileDataLayer wmsProjectionLayer;
+        public WMSTileDataLayer WMSProjectionLayer
+        {
+            get
+            {
+                if (wmsProjectionLayer == null)
+                    wmsProjectionLayer = GetComponent<WMSTileDataLayer>();
+                return wmsProjectionLayer;
+            }
+        }
+
+        protected LayerURLPropertyData urlPropertyData = new();
+        LayerPropertyData ILayerWithPropertyData.PropertyData => urlPropertyData;
+
+        public bool TransparencyEnabled { get => WMSProjectionLayer.TransparencyEnabled; }
+        public int DefaultEnabledLayersMax { get => WMSProjectionLayer.DefaultEnabledLayersMax; }
+        public Vector2Int PreferredImageSize { get => WMSProjectionLayer.PreferredImageSize; }
+
+        protected override void Awake() 
+        {
+            base.Awake();
+            wmsProjectionLayer = GetComponent<WMSTileDataLayer>();
+            LayerData.RootIndexChanged.AddListener(SetRenderOrder);
+        }
+
+        public void SetURL(string url)
+        {
+            this.urlPropertyData.url = url;
+            wmsProjectionLayer.WmsUrl = url;
```

Comment:
Zie https://github.com/Netherlands3D/twin/pull/288/files#r1799081292; en is SetURL nodig; of is dat eigenlijk een alias voor het assignen van urlPropertyData.url

Reply:
Deze snap ik niet, de url moet toch gezet worden vanuit de adapter?